src/hotspot/share/runtime/os.cpp

```diff
@@ -736,7 +736,7 @@ void  os::free(void *memblock) {
   // If NMT is enabled, this checks for heap overwrites, then de-accounts the old block.
   void* const old_outer_ptr = MemTracker::record_free(memblock);
 
-  ::free(old_outer_ptr);
+  ALLOW_C_FUNCTION(::realloc, ::free(old_outer_ptr);)
```

Comment:
s/realloc/free/ - unfortunately, the existing macro implementations only use the name for "documentation" purposes.

Reply:
Fixed